addons/sale_project/static/src/components/project_right_side_panel/components/project_profitability_section.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <templates id="template" xml:space="preserve">
     <t t-name="sale_project.ProjectProfitabilitySection">
-        <tr>
+        <tr t-att-id="revenue.id">
```

Comment:
I am using it in the persistence patch to specifically target the revenue section rather than sequence as sequence order can be changed by disabling or enabling some settings.